ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -548,3 +522,36 @@ func getClusterSize(context *cli.Context) (string, error) {
 
 	return size, nil
 }
+
+// If param1 exists, param2 is not allowed.
+func validateMutuallyExclusiveParams(cfnParams *cloudformation.CfnStackParams, param1, param2 string) bool {
+	if _, err := cfnParams.GetParameter(param1); err != nil {
+		return false
+	}
+	if _, err := cfnParams.GetParameter(param2); err != cloudformation.ParameterNotFoundError {
+		return true
+	}
+	return false
+}
+
+// If param1 exists, param2 is required.
+func validateDependentParams(cfnParams *cloudformation.CfnStackParams, param1, param2 string) bool {
```

Comment:
same comment as above - suggested renaming to "assertDependentParamMissing"

Reply:
see above